src/services/data/DataService.tsx

```diff
@@ -7,11 +8,16 @@ export class DataService {
   static postMultipartRequest(data: any, fullURI?: string): Promise<any> {
     const formData = new FormData();
     if (fullURI === undefined) {
-      fullURI = process.env.REACT_APP_MCP_DATA_SOURCE + "files";
+      fullURI =
+        process.env.REACT_APP_MCP_DATA_MAPPING_UTIL + "api/excel/jsontoexcel";
```

Comment:
The name of this method is very generic, which implies that it can be used for many purposes, but there is a default URI that makes the default a very specific purpose. This doesn't quite feel right to me. I'm open to argument otherwise, but my gut reaction is to rename this method as something more specific (i.e. `uploadAndValidateFile`) and not accept an optional fullURI parameter. Thoughts?

Reply:
@orndorffgrant We could and maybe we should. Tell you what, I am going to spin off a couple of much smaller tickets before I merge this and I will change it then to reflect this.